pkg/logs/config/integration_config.go

```diff
@@ -71,6 +71,10 @@ type LogsConfig struct {
 	AutoMultiLine               *bool   `mapstructure:"auto_multi_line_detection" json:"auto_multi_line_detection"`
 	AutoMultiLineSampleSize     int     `mapstructure:"auto_multi_line_sample_size" json:"auto_multi_line_sample_size"`
 	AutoMultiLineMatchThreshold float64 `mapstructure:"auto_multi_line_match_threshold" json:"auto_multi_line_match_threshold"`
+
+	// For Type="file" when used for logging containers, this gives the container runtime, allowing
```

Comment:
NIT: maybe we can reword this a little:

```
When logging containers and Type="file" this is set to the current container runtime...
```

I had to read this a few times to understand what it was saying (could just be me)

Reply:
Yeah, that sounds good!